src/components/atoms/ContainerWithBackground/ContainerWithBackground.tsx

```diff
@@ -0,0 +1,47 @@
+import { useCss } from "react-use";
+import classNames from "classnames";
+
+import { DEFAULT_VENUE_BANNER } from "settings";
+
+import { AnyVenue, EmbeddableVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import { useValidImage } from "hooks/useCheckImage";
+
+type ContainerWithBackgroundProps = {
+  containerName: string;
+  className?: string;
+  venue: WithId<AnyVenue> | EmbeddableVenue;
+  style?: React.CSSProperties;
+};
+
+export const ContainerWithBackground: React.FC<ContainerWithBackgroundProps> = ({
```

Comment:
looks like it's not a Container with background, but Venue with Background, because there is a `venue` props.
If you want to keep it "Container", then you' should better specify "backgroundImage" props instead of venue.

Reply:
renamed it to VenueWithOverlay in b396ea2